specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json

```diff
@@ -369,8 +369,12 @@
       ],
       "properties": {
         "name": {
-          "description": "The text line style name.",
+          "description": "The text line style name, including handwriting and other.",
           "type": "string",
+          "x-ms-enum": {
+            "name": "name",
```

Comment:
This name will be used as the default class/enum name when auto-generating the SDK.  Consider "TextStyle" instead.

Reply:
In our json result of read/layout, we have 'name' as key, this is what we delivery for Layout 2.1-preview.2

Reply:
Thanks for reminder, I have updated the name.